src/dotnet/Core/Links.cs

```diff
@@ -24,6 +24,9 @@ public static LocalUrl EmbeddedChat(ChatId chatId, long? entryId = null)
             ? $"/embedded/{chatId.Value}" + ChatEntryQuery(vEntryId)
             : $"/embedded/{chatId.Value}";
 
+    public static LocalUrl PlaceInfo(PlaceId placeId)
+        => $"/place-info/{placeId.Value}";
```

Comment:
is `/place/...` taken or supposed to be taken by some other page?

Reply:
Done